modules/src/erc721/extensions/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod erc721metadata;
```

Comment:
Clippy is not complaining about missing underscore? I'd change it to `erc721_metadata` anyway.

Reply:
Done.